### ISSUE TITLE
Accessibility: Select box has no description

### ISSUE THREAD

**Issue by eserkansozer** [corejavascript/typeahead.js#240]
Version: 1.3.1
Browser: Chrome
Plugin to run: Siteimprove

It seems there is an accessibility issue which prevents WCAG 2 A standard compatibility.

Error description from Siteimprove accessibility testing tool report based on WCAG 2:

"Drop down menus (select boxes or role="listbox") should always have a description that is explicitly associated with the field to make sure that users of assistive technologies will also know what the field is for.

If the drop down menu (select box or role="listbox") has a visible description indicating the purpose of the menu, this description should be explicitly associated to the input drop down menu either as a HTML label or using the WAI-ARIA attribute 'aria-labelledby'.

If it is not possible to add a visible description, either add a mouseover text ('title' attribute) to the drop down menu or create an invisible label using the WAI-ARIA attribute 'aria-label'."

An aria-labelledby attribute should be added to the div with role="listbox" to fix the accessibility issue. This may be an optional parameter in initialization.
